DataFormats/simulation/include/SimulationDataFormat/MCTrack.h

```diff
@@ -226,12 +234,14 @@ class MCTrackT
     struct {
       int storage : 1;  // encoding whether to store this track to the output
       int process : 6;  // encoding process that created this track (enough to store TMCProcess from ROOT)
-      int hitmask : 24; // encoding hits per detector
+      int hitmask : 18; // encoding hits per detector (we have 18 detectors)
```

Comment:
we need to be careful here. This may currently work, but with the addition of new modules (such as ALICE3) we might quickly overrun this.  (We could introduce another mapping of what detectors are actually present at runtime and save this information persistently). It's ok for now. I'll open a ticket to follow up on this.

Reply:
I see. ALICE3 might need an unknown number of bits... so one might even overflow with the current 32 bit scheme.
But, I think that when running in ALICE3 setup the meaning of the bits in the `hitmask` will be translated differently.

Anyway, let me then put 4 bits reserved for simulation bits (`tobedone`, `inhibit`, `reserved1`, `reserved2`) and increase the  `hitmask` to 21.

Reply:
Yes we can have a translation of bits. But this also needs to be saved and requires space in the ROOT file.